pom.xml

```diff
@@ -69,13 +102,18 @@
         <dependency>
             <groupId>org.spigotmc</groupId>
             <artifactId>spigot-api</artifactId>
-            <version>1.17.1-R0.1-SNAPSHOT</version>
+            <version>1.18.1-R0.1-SNAPSHOT</version>
             <scope>provided</scope>
         </dependency>
         <dependency>
-            <groupId>preponderous</groupId>
-            <artifactId>ponder</artifactId>
-            <version>v0.14-alpha-2</version>
+            <groupId>com.github.Preponderous-Software</groupId>
+            <artifactId>Ponder</artifactId>
+            <version>v1.0</version>
```

Comment:
This should be changed to 1.0. It should match the JAR.

Reply:
ah, i use jitpack so it v1.0

Reply:
Sounds good. Would you mind adding the jitpack information to the README?